h2o-mapreduce-generic/src/main/java/water/hadoop/FSCloudingEventSource.java

```diff
@@ -0,0 +1,150 @@
+package water.hadoop;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+
+import java.io.*;
+import java.util.*;
+import java.util.stream.Collectors;
+
+class FSCloudingEventSource {
```

Comment:
since we already have a class FileSystemBasedClouding I would prefer calling this FileSystemCloudingEventSource

Reply:
good idea